blahablaha/foo

```diff
@@ -0,0 +1 @@
+foo content
```

Comment:
Can `blahablaha/foo` and `immer/bar` be removed now? Looks like the test is using `/tmp`.